pkg/kube/client.go

```diff
@@ -275,10 +275,16 @@ func (c *Client) WatchUntilReady(namespace string, reader io.Reader) error {
 	return perform(c, namespace, reader, watchUntilReady)
 }
 
-const includeThirdPartyAPIs = false
+// IncludeThirdPartyAPIs indicates whether to load "dynamic" APIs.
+//
+// This requires additional calls to the Kubernetes API server, and these calls
+// are not supported by all versions. Additionally, during testing, initializing
+// a client will still attempt to contact a live server. In these situations,
+// this flag may need to be disabled.
+var IncludeThirdPartyAPIs = true
```

Comment:
What do you think about adding this as a field on `Client`?  That would remove the global var and the hacky init() call.